Example/Example/AnimationListView.swift

```diff
@@ -0,0 +1,93 @@
+// Created by Cal Stephens on 7/11/23.
+// Copyright © 2023 Airbnb Inc. All rights reserved.
+
+import Lottie
+import SwiftUI
+
+// MARK: - AnimationListView
+
+struct AnimationListView: View {
+
+  let directory: String
+
+  var body: some View {
```

Comment:
Nice and simple -- this was like 200 LOC plus the entire EpoxyCollectionView / EpoxyLayoutGroups dependency before in the UIKit example app